src/Umbraco.Community.Sustainability/Models/SustainabilityData.cs

```diff
@@ -2,65 +2,65 @@ namespace Umbraco.Community.Sustainability.Models
+        public string? carbonRating { get; set; }
+        public Emissions? emissions { get; set; }
+        public Resource[]? resources { get; set; }
     }
 
     public record Emissions
     {
-        public float co2 { get; set; }
+        public double? co2 { get; set; }
         public bool green { get; set; }
-        public Variables variables { get; set; }
+        public Variables? variables { get; set; }
     }
 
     public record Variables
     {
-        public string description { get; set; }
-        public int bytes { get; set; }
-        public Gridintensity gridIntensity { get; set; }
-        public float dataReloadRatio { get; set; }
-        public float firstVisitPercentage { get; set; }
-        public float returnVisitPercentage { get; set; }
+        public string? description { get; set; }
+        public int? bytes { get; set; }
+        public GridIntensity? gridIntensity { get; set; }
+        public double? dataReloadRatio { get; set; }
+        public double? firstVisitPercentage { get; set; }
+        public double? returnVisitPercentage { get; set; }
     }
 
-    public record Gridintensity
+    public record GridIntensity
     {
-        public string description { get; set; }
-        public float network { get; set; }
-        public float dataCenter { get; set; }
-        public float production { get; set; }
-        public float device { get; set; }
+        public string? description { get; set; }
+        public double? network { get; set; }
+        public double? dataCenter { get; set; }
+        public double? production { get; set; }
+        public double? device { get; set; }
     }
 
     public record Resource
     {
-        public string name { get; set; }
-        public string entryType { get; set; }
-        public float startTime { get; set; }
-        public float duration { get; set; }
-        public string initiatorType { get; set; }
-        public string deliveryType { get; set; }
-        public string nextHopProtocol { get; set; }
-        public string renderBlockingStatus { get; set; }
-        public int workerStart { get; set; }
-        public int redirectStart { get; set; }
-        public int redirectEnd { get; set; }
-        public float fetchStart { get; set; }
-        public float domainLookupStart { get; set; }
-        public float domainLookupEnd { get; set; }
-        public float connectStart { get; set; }
-        public float secureConnectionStart { get; set; }
-        public float connectEnd { get; set; }
-        public float requestStart { get; set; }
-        public float responseStart { get; set; }
-        public int firstInterimResponseStart { get; set; }
-        public float responseEnd { get; set; }
-        public int transferSize { get; set; }
-        public int encodedBodySize { get; set; }
-        public int decodedBodySize { get; set; }
-        public int responseStatus { get; set; }
+        public string? name { get; set; }
```

Comment:
not sure we still need all these props if we're not using them. suggest we trim the model to include the props we actively use.

this will make future versions a bit easier to maintain as well, as co2.js changed their response to account for the updated sustainable web design ratings and calculations.

 https://www.thegreenwebfoundation.org/news/release-guide-co2-js-v0-16/